src/99-appendices/11-qmri.md

```diff
@@ -0,0 +1,558 @@
+           "anat/sub-01_fa-4_VFA.nii.gz",
+           "fmap/sub-01_TB1map.nii.gz"],
+"EstimationPaper":"Deoni et. al.MRM, 2015",
+"EstimationAlgorithm":"Linear",
+"Units": "second",
+
+<<Parameters that are constant across files in the (parent) file collection>>
+
+"MagneticFieldStrength”: "3”,
+"Manufacturer”: "Siemens”,
+"ManufacturerModelName”: "TrioTim”,
+"InstitutionName”: "xxx”,
+"PulseSequenceType”: "SPGR”,
+"PulseSequenceDetails”: "Information beyond the sequence type that identifies
+ specific pulse sequence used (VB version, if not standard, Siemens WIP XXX
+ ersion ### sequence written by xx using a version compiled on mm/dd/yyyy/)”,
+"RepetitionTimeExcitation": "35",
+"EchoTime": "2.86",
+"SliceThickness": "5",
+
+<<Relevant parameters that vary across the linking entity of the (parent) file collection>>
+
+"FlipAngle": ["5","10","15","20"]
+
+}
+```
+
+## Deriving the intended qMRI application from an ambiguous file collection
+
+Certain file colleciton suffixes may refer to a generic data collection regime such as variable flip angle (VFA),
+rather than a more specific acquisition, for example, magnetization prepared two gradient echoes (MP2RAGE).
+Such generic acquisitions can serve as a basis to derive various qMRI applications by changes to the acquisition sequence
+(for example, readout) type or by varying additional scan parameters.
+
+If such an inheritance relationship is applicable between an already existing file collection
+and a new qMRI application to be included in the specification,
+the inheritor qMRI method is listed in the table below instead of introducing a new file collection suffix.
+This approach aims at:
+
+-   preventing the list of available suffixes from over-proliferation,
+-   providing qMRI-focused BIDS applications with a set of meta-data driven rules to infer possible fitting options,
+-   keeping an inheritance track of the qMRI methods described within the specification.
+
+| **File-collection suffix** | **If REQUIRED metadata == Value** | **OPTIONAL metadata (`entity`/`fixed`)**   | **Derived application name (NOT a suffix)** |
+|----------------------------|-----------------------------------|--------------------------------------------|---------------------------------------------|
+| VFA                        | `PulseSequenceType` == `SPGR`     |                                            | DESPOT1                                     |
+| VFA                        | `PulseSequenceType` == `SSFP`     | `SpoilingRFPhaseIncrement` (`fixed`)       | DESPOT2                                     |
+| MP2RAGE                    |                                   | `EchoTime` (`echo`)                        | MP2RAGE-ME                                  |
+| MPM                        |                                   | `EchoTime` (`echo`)                        | MPM-ME                                      |
+
+<sup>*</sup> (`entity`/`fixed`) denotes whether the OPTIONAL metadata that forms a new
```

Comment:
I'm assuming the `<sup>*</sup>` was missing in the table header because it caused formatting issues?

```suggestion
In this table, (`entity`/`fixed`) denotes whether the OPTIONAL metadata that forms a new
```